packages/server/modules/workspaces/tests/integration/repositories.spec.ts

```diff
@@ -281,4 +283,61 @@ describe('Workspace repositories', () => {
       await expectToThrow(() => upsertWorkspaceRole(role))
     })
   })
+
+  describe.only('getWorkspaceRolesCountFactory creates a function, that', () => {
+    it('returns an empty array with no users in workspace', async () => {
+      expect(
+        await getWorkspaceRolesCountFactory({ db })({
+          workspaceId: createRandomPassword()
+        })
+      ).to.deep.eq([])
+    })
+    it('returns an array containing counts for each role', async () => {
+      const { id: userId1 } = await createAndStoreTestUser()
```

Comment:
1. There's a `createTestUsers` test util for batching up these calls
2. Or do Promise.all()?